.github/renovate.json

```diff
@@ -8,9 +8,6 @@
   "schedule": [
     "monthly"
   ],
```

Comment:
FYI the [Renovate instructions](https://docs.renovatebot.com/configuration-options/#vulnerabilityalerts) say that we should enable Dependabot alerts in our repo settings. I will do this when I complete the PR (won't turn on Dependabot updates though).

https://github.com/ni/nimble/settings/security_analysis


![image](https://github.com/ni/nimble/assets/10500124/4182e063-154b-43bb-b295-3f5a428ac9f2)